recipe/meta.yaml

```diff
@@ -1,119 +1,121 @@
-    - "$RPATH/torch_cpu.dll"       # [win] 
+    - "**/libc10.so"              # [linux]
+    - "**/libtorch_cpu.so"        # [linux]
+    - "**/libc10_cuda.so"         # [cuda_compiler_version != "None" and linux]
+    - "**/libtorch_cuda.so"       # [cuda_compiler_version != "None" and linux]
+    - "**/ld64.so.1"              # [s390x]
+    - "**/libc10.dylib"           # [osx]
+    - "**/libtorch_cpu.dylib"     # [osx]
+    - "**/c10.dll"                # [win]
+    - "**/torch_cpu.dll"          # [win]
 
 requirements:
   build:
+    #- {{ stdlib('c') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    # libpng and libjpeg must be available at compilation time in order to be available,
-    # see https://github.com/pytorch/vision/tree/v0.11.3#image-backend
-    - jpeg      # [not win]
-    - libpng
+    {% if cuda_major >= 12 %}
+    - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
+    - libcublas-dev                          # [build_platform != target_platform]
+    - libcusolver-dev                        # [build_platform != target_platform]
+    - libcusparse-dev                        # [build_platform != target_platform]
+    {% endif %}
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - pytorch ={{ compatible_pytorch }}.*={{ torch_proc_type }}*  # [build_platform != target_platform]
   host:
     - python
     - pip
+    - setuptools
     - wheel
-    - numpy {{ numpy }}
-    - pytorch 2.0.1
-    # libpng and libjpeg must be available at compilation time
-    - jpeg {{ jpeg }}  # [not win]
+    - cudnn                                      # [cuda_compiler_version != "None"]
+    {% if cuda_major >= 12 %}                    # [cuda_compiler_version != "None"]
+    - libcublas-dev                              # [cuda_compiler_version != "None"]
+    - libcusolver-dev                            # [cuda_compiler_version != "None"]
+    - libcusparse-dev                            # [cuda_compiler_version != "None"]
+    {% endif %}                                  # [cuda_compiler_version != "None"]
+    - cudatoolkit {{ cuda_compiler_version }}    # [cuda_compiler_version == "11.8"]
+    - cuda-version {{ cuda_compiler_version }}   # [cuda_compiler_version != "None"]
+    # split off image/video into separate outputs?
+    - jpeg {{ jpeg }}
     - libpng {{ libpng }}
-    # GPU requirements
-    - cudatoolkit {{ cudatoolkit }}  # [pytorch_variant == "gpu"]
-    # 2022/08/25: Not requiring ffmpeg because torchvision has bugs with certain ffmpeg versions, 
-    # see https://github.com/pytorch/vision/issues/5419#issuecomment-1193483864
-    # and https://github.com/conda-forge/torchvision-feedstock/commit/4ec4b5f4eb4889dbb1f8da34662ea622bb4b3828
-    # - ffmpeg
+    - ffmpeg
+    - numpy
```

Comment:
numpy is needed only at runtime, right? See https://github.com/pytorch/vision/blob/v0.18.0/pyproject.toml#L18 and https://github.com/pytorch/vision/blob/v0.18.0/setup.py#L60-L63

Reply:
done